yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
+    definition: Lattice translation vectors lengths (a, b, c) of an atomic unit cell
+    shape: [3]
+    type: double
+
+  LATTICE_UC_ANGLES:
+    definition: Alpha, Beta and Gamma angles of an atomic unit cell
+    shape: [3]
+    type: double
+
+  SYMMETRY_GROUP:
+    definition: Symmetry group number acording to the Volume A of International Tables for Crystallography
+    shape: [1]
+    type: integer
+
+  CRYSTAL_ORIENTATION_1:
+    definition: Crystallographic orientation defined by Miller Index of the reference surface
+    shape: [3, 3]
+    type: double
+
+  CRYSTAL_ORIENTATION_2:
+    definition: Crystallographic orientation defined by Miller Index of the non-reference surface
+    shape: [3, 3]
+    type: double
+
```

Comment:
Indeed you are right, there is only one crystallographic orientation. These two are different orientations: one is the crystallographic orientations as we know its (how the replicated cells are going to be orientated along the component shape) and the other is the orientation of the full shape in the global 3D axis. The shape is 3x3 as these orientations define two new 3D references to orientate the building process of the component and the position of it
What I agree is that maybe the names are not really that clear, we will think about this